src/imitation/util/rollout.py

```diff
@@ -51,6 +52,13 @@ def unwrap_traj(traj: Trajectory) -> Trajectory:
   return res
 
 
+def rewrite_rewards_traj(traj: Trajectory, reward_fn: RewardFn) -> Trajectory:
```

Comment:
Per my comment above not sure if this is the right atomic operation, or if we should just return the `new_rews`.